src/backend/utils.ts

```diff
@@ -915,19 +936,79 @@ async function ContinueWithFoundWine(
   return { response }
 }
 
+export async function isRosettaAvailable() {
```

Comment:
non-blocker: return false instead of undefined

Reply:
changed the name of the method